src/main/java/com/amihaiemil/versioneye/MkJsonServer.java

```diff
@@ -79,6 +70,9 @@ private static void initServer(final NfJsonObjectBuilder storage) {
                 .add("success", true)
                 .add("message", "pong")
                 .build()
+        ).add(
+            "authenticated",
+            Json.createArrayBuilder().build()
         );
```

Comment:
@SherifWaly I initialized the server with an empty array for authenticated users and then we get rid of that ugly null check.